### ISSUE TITLE
A11y - Override for D7 Bootstrap Accessbility Collapse

### ISSUE THREAD

**Issue by joewhitsitt** [uiowa/uiowa#518]
## Problem
Across our D7 sites, we have implemented PayPal's Bootstrap Accessibility Plugin to make Bootstrap 3 more accessible.

For the Collapse component which is used by Accordions, PayPal's JS assumes there is an 'in' class which opens one of the accordion items by default. Without it, the accordion is no keyboard accessible.

https://github.com/paypal/bootstrap-accessibility-plugin/issues/121

Some sites may use the Bootstrap Accessibility module which pulls directly from PayPal's repo.

## Proposed Solution
Since the plugin hasn't been updated in years and the issue itself has stalled, I suggest fixing the related snippet for all the sites that use it.

The problem is that this not served from a central location and is local to about 15-20 sites:
https://github.com/search?p=1&q=org%3AITS-UofIowa+%22collpanel.attr%28%27id%27%29%2C+%27tabindex%27%3A%27-1%22&type=Code

Additionally, we are using the Bootstrap Accessibility module on some sites so we should investigate how we are adding the library for this: https://www.drupal.org/project/bootstrap_accessibility

Replace:
`colltab.attr({'aria-controls' : collpanel.attr('id'), 'tabindex':'-1' })`
With:
`colltab.attr({'aria-controls' : collpanel.attr('id'), 'tabindex':'0' })`

on the site's master branch, create a release for each site and deploy the changes to the production environments.

## Test
Test one site locally before doing this across all sites. This came up as an issue on the ITS website: https://its.uiowa.edu/support/article/111136

A user should be able to tab through the links on the page, including the accordion items. Focused on an accordion item header, the user should be able to hit enter to expand/collapse.

**Comment by joewhitsitt** [comment]
All but catering and honors have at least had the fix merged into master. Whether a release has been created and pushed out is another story. Closing
